guides/source/3_1_release_notes.md

```diff
@@ -102,7 +102,7 @@ gem "jquery-rails"
     # config.assets.precompile `= %w( admin.js admin.css )
 
 
-    # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
+    # Force all access to the app over TLS/SSL, use Strict-Transport-Security, and use secure cookies.
```

Comment:
We don't need to update release notes from 15 years ago

Reply:
Agreed. I reverted this file in the latest revision. Thanks for the review.